packages/x-charts/src/ChartsLabel/ChartsLabelMark.tsx

```diff
@@ -6,12 +6,18 @@ import clsx from 'clsx';
 import { ChartsLabelMarkClasses, labelMarkClasses, useUtilityClasses } from './labelMarkClasses';
 import { consumeThemeProps } from '../internals/consumeThemeProps';
 
+export interface ChartsLabelCustomMarkProps {
+  className?: string;
+  /** Color of the series this mark refers to. */
```

Comment:
Could be better to mention https://github.com/mui/mui-x/pull/16739/files#r1971331114 here instead

Reply:
Not sure, actually 😅 I wrote it in the docs, but let me know if you think I should move it here